[2025-07-20T02:24:02.226+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-20T02:24:02.260+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_weather_forecast.fetch_forecast scheduled__2025-07-19T18:00:00+00:00 [queued]>
[2025-07-20T02:24:02.267+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_weather_forecast.fetch_forecast scheduled__2025-07-19T18:00:00+00:00 [queued]>
[2025-07-20T02:24:02.268+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-20T02:24:02.280+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_forecast> on 2025-07-19 18:00:00+00:00
[2025-07-20T02:24:02.290+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'fetch_weather_forecast', 'fetch_forecast', 'scheduled__2025-07-19T18:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/fetch_weather_data.py', '--cfg-path', '/tmp/tmp56rxfzo6']
[2025-07-20T02:24:02.292+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask fetch_forecast
[2025-07-20T02:24:02.293+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=176) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-20T02:24:02.294+0000] {standard_task_runner.py:63} INFO - Started process 230 to run task
[2025-07-20T02:24:02.345+0000] {task_command.py:426} INFO - Running <TaskInstance: fetch_weather_forecast.fetch_forecast scheduled__2025-07-19T18:00:00+00:00 [running]> on host 0189e20d7959
[2025-07-20T02:24:02.441+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_weather_forecast' AIRFLOW_CTX_TASK_ID='fetch_forecast' AIRFLOW_CTX_EXECUTION_DATE='2025-07-19T18:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-19T18:00:00+00:00'
[2025-07-20T02:24:02.442+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-20T02:24:13.053+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-20T02:24:13.054+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-20T02:24:13.066+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=fetch_weather_forecast, task_id=fetch_forecast, run_id=scheduled__2025-07-19T18:00:00+00:00, execution_date=20250719T180000, start_date=20250720T022402, end_date=20250720T022413
[2025-07-20T02:24:13.107+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-20T02:24:13.124+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-20T02:24:13.127+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
